hw/ip/otp_ctrl/data/BUILD

```diff
@@ -14,7 +14,7 @@ autogen_hjson_header(
 )
 
 otp_image(
-    name = "rma_image_verilator",
+    name = "test_rma_image",
```

Comment:
I'm not sure we want to name this "test_rma_image" in the long run because we use there are other "test" lifecycle states this could be confused with. There is only one RMA lifecycle state. However, I'm ok with punting this to future a change (keeping as is for now) as there is currently no naming convention for OTP images, for which the lifecycle bits are just a small portion of. (I.e., we may want to include other bits of information about the generated OTP image in its name but we will need to settle on the convention first)

Reply:
There seems to be a defacto standard among the filenames: `otp_ctrl_img_<lifecycle-state>.hjson`.

Maybe we should just go with `img_<lifecycle-state>` as the bazel label names for now.

Reply:
sgtm!